common/views/components/ExhibitionHeader/README.md

```diff
@@ -0,0 +1,3 @@
+## Exhibition header
+
+TODO: add purpose of exhibition header.
```

Comment:
@Heesoomoon @GarethOrmerod maybe something to talk about during the header meeting, it might be nice to flag up why it is bigger / image led etc.